_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,289 @@
+
+```bash
+kubectl expose deployment laravel-kubernetes-demo --type=NodePort --port=8181
+```
+
+and provided all went well, you will see a confirmation similar to:
+
+```bash
+service "laravel-kubernetes-demo" exposed
+```
+Running the following command:
+
+```bash
+kubectl get services
+```
+will show you a list of running services. You can also view the running service under the "Services" navigation menu within the dashboard. A more exciting way to verify this deployment and the service exposure is obviously seeing the running application in the browser 😊
+
+To obtain the URL of the application (service), you can use the following command:
+
+```bash
+minikube service --url=true laravel-kubernetes-demo
+```
+
+which will output the IP address and port number similar to:
+
+```bash
+http://192.168.99.101:31399
+```
+
+or, launch the application directly in the browser:
+
+```bash
+minikube service laravel-kubernetes-demo
+```
+
+__Scaling__
+
+And that is it. You have successfully deployed the application in Kubernetes. It's exciting. But what's the point of doing all of this? Well, you only have one deployment with a single Pod running, provisioned to a Node with the exposed web service. Let's replicate this pod to two more pods acting as load balancers to the web traffic.
+
+So that you understand where you are at this moment, run the following command to get a list of desired and available Pods:
+
+```bash
+kubectl get deployment
+
+NAME                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+laravel-kubernetes-demo   1         1         1            1           57m
+```
+
+The output will be "1" for each. You want to have three available Pods so let's scale this up:
+
+```bash
+kubectl scale --replicas=3 deployment/laravel-kubernetes-demo
+deployment "laravel-kubernetes-demo" scaled
+```
+
+Done. You have replicated the first Pod to another two, giving you three Pods running this service. Running the `get deployment` command will verify this.
+
+```bash
+kubectl get deployment
+
+NAME                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+laravel-kubernetes-demo   3         3         3            3           59m
+```
+
+You can also see this in the Dashboard under Pods or in the Service detail screen.
+
+Now the application's deployment services are running on three 
+separate, replicated Pods.
```

Comment:
I think this is an opportunity to explain why Kubernetes is cool. We could add:

Imagine your application becoming even more popular. Thousands of visitors are flocking to your website. In the past you may have been busy launching more scripts to create more instances of your application. In Kubernetes you can scale to ten instances in a snap:

```bash
kubectl scale --replicas=10 deployment/laravel-kubernetes-demo
deployment "laravel-kubernetes-demo" scaled
```

You can see how convenient it is to use Kubernetes to scale your website.

Reply:
Perfect 👍 I like it.